themes/default/css/_besocial/index_besocial.css

```diff
@@ -2066,6 +2066,9 @@ a.help .icon {
 		box-shadow: 3px 3px 3px rgba(0, 0, 0, 0.3);
 	}
 
+	#main_menu .linklevel1.active, #main_menu .linklevel1.active:hover {
+		border: 0px;
```

Comment:
border: 0; or border: none; would be preferable to try to cut down on px instances.